test/E2E/EndToEndTest.cs

```diff
@@ -9,110 +9,120 @@
 using Xunit;
 using Microsoft.DotNet.Cli.Utils;
 using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.DotNet.Tools.Publish.Tests;
 
-namespace ConsoleApplication
+namespace Microsoft.DotNet.Tests.EndToEnd
 {
-    public class E2ETest
+    public class EndToEndTest : TestBase
     {
         private static readonly string EXPECTED_OUTPUT = "Hello World!" + Environment.NewLine;
-        private static readonly string TESTDIR_NAME = "hellotest";
+        private static readonly string TESTDIR_NAME = "e2etestroot";
         private static readonly string OUTPUTDIR_NAME = "testbin";
-
-        private static string RootPath { get; set; }
+        
+        private string Rid { get; set; }
         private string TestDirectory { get; set; }
```

Comment:
I am still sleepy from my vacation. Just change the static readonly fields to `s_*`.